pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -277,6 +277,19 @@ func (ds *dockerService) StopPodSandbox(ctx context.Context, r *runtimeapi.StopP
 	}
 
 	// TODO: Stop all running containers in the sandbox.
+	go func() {
+		containers, err := ds.client.ListContainers()
+		if err != nil {
+			klog.Errorf("Failed to list all container: %v", err)
+			return
+		}
+		for _, v := range containers {
+			err := ds.client.StartContainer(v.ID, defaultSandboxGracePeriod)
```

Comment:
should this be StopContainer?

Reply:
Yeah.It has called docker service interface.

Reply:
Ref. https://github.com/kubernetes/kubernetes/blob/a73730147c8b45ad37a56ba210a9c721ed7a6844/pkg/kubelet/dockershim/docker_sandbox.go#L264